File: 2020/helpers/helpers_test.py
import unittest

from advent_of_code.helpers.helpers import read_file, read_file_line_by_line

class TestHelpers(unittest.TestCase):
    def test_read_file(self) -> None:
        self.assertEqual(
            read_file(filepath="tests/helpers/helpers_test.txt"),
            "this\nis\na\ntest\n"
        )
    def test_read_file_line_by_line(self) -> None:
        self.assertEqual(
            read_file_line_by_line(filepath="tests/helpers/helpers_test.txt"),
            ["this", "is", "a", "test"]
        )

File: 2020/helpers/helpers_test.java
package tests.helpers;

import advent_of_code.helpers.Helpers;

import static org.junit.jupiter.api.Assertions.assertEquals;

import java.util.ArrayList;

import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;



public class helpers_test {

    @Test                                               
    @DisplayName("ReadFileLineByLine")   
    public void testReadFileLineByLine() throws Exception {

        ArrayList<String> expectedResult = new ArrayList<String>() { 
            { 
                add("this"); 
                add("is"); 
                add("a"); 
                add("test"); 
            } 
        }; 
        ArrayList <String> calculatedOutput = Helpers.readFileLineByLine("tests/helpers/helpers_test.txt");

        assertEquals(expectedResult, calculatedOutput);          
    }

    @Test                                               
    @DisplayName("readfile") 
    public void testReadFile() throws Exception {
        String expectedResult = "this\nis\na\ntest\n";
        String calculatedOutput = Helpers.readFile("tests/helpers/helpers_test.txt");
        assertEquals(expectedResult, calculatedOutput);          
    }
    
}